blog/fine-grained-news-2025-09.md

```diff
@@ -0,0 +1,114 @@
+<!-- markdown-link-check-enable -->
+
+## New OpenFGA Adopters
+
+The list of companies/projects keeps increasing! These were added since the last edition of Fine Grained News:
+
+- [Headspace](https://www.headspace.com/join-us): Headspace uses OpenFGA to manage entitlements for its users based on their subscriptions. OpenFGA is also used to determine availability of features and content based on regionality and language. 
+- [EarthScope Consortium](https://www.earthscope.org/): EarthScope Consortium supports transformative global geophysical research and education. They leverage OpenFGA to authorize researcher access to hundreds of thousands of data streams from geophysical sensors located all over the world. 
+- [Incus](https://linuxcontainers.org/incus/): The Incus project uses OpenFGA as its primary authorization mechanism for fine grained access control to all its resources. 
+- [virv.ai](https://virv.ai): virv.ai uses OpenFGA as the core authorization model that enables individuals, agents and organizations to access artifacts such as data, credentials, and documents. OpenFGA allows us to meet the needs of the AI-enabled Enterprise where trust is essential. 
+- [X-HR LABS](https://x-hr.co): X-HR is a modern HR tech platform that empowers businesses with full freedom to use and grow on their own terms. They leverage OpenFGA to implement robust, relationship-based access control across company data and organizational structures. 
+- [AppsCode](https://appscode.com): AppsCode uses OpenFGA to implement authorization functionality for their AppsCode Container Engine (ACE) Platform.
+- [Grafana Labs](https://grafana.com/): Grafana user authorization and Role Based Access Control (RBAC) are migrating to OpenFGA. 
+
+You can also learn how OpenFGA powers [NeoNephos](https://neonephos.org/), a new Linux Foundation project that's dedicated to advancing open source projects that align with the strategic objectives of the EU's IPCEI-CIS, in [this Open Source Summit](https://www.youtube.com/watch?v=vpDGQgCaLt8) presentation by [Bastian Echterhölter](https://www.linkedin.com/in/bastianechterhoelter/) & [Aaron Schweig](https://www.linkedin.com/in/aaron-schweig-471707199/) from SAP.
+
+[Nagarro](https://www.nagarro.com/) published [a case study](https://www.linkedin.com/posts/nagarro_schneider-success-story-activity-7366395183585763329-up7m/) detailing how they helped [Schneider Electric](https://www.se.com/) with an OpenFGA implementation.
+
+The [Linux Foundation](https://www.linuxfoundation.org/) is using OpenFGA for their new platform. You can see their OpenFGA authorization model [here](https://github.com/linuxfoundation/lfx-v2-helm/blob/f40018f3031ad106f52cfb29b3f9103821ad63b5/charts/lfx-platform/templates/openfga/model.yaml#L25).
```

Comment:
There's a reference to Linux Foundation, Nagarro, then Linux Foundation again. If we can make the LF ones closer together, that would flow better